exercise-observer/controllers/heading.controller.js

```diff
@@ -0,0 +1,29 @@
+class HeadingController {
+  constructor(model) {
+    this.model = model;
+  }
+
+  //EVENTLISTENER INTERFACE
```

Comment:
Please reference best practices for comment in JS at
https://blog.bitsrc.io/best-practices-for-using-comments-in-javascript-4c4cd8619c18